components/integrations/bots/index.ts

```diff
@@ -59,6 +66,7 @@ type Actions = {
 function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
         actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc>, Actions>({
+            getAppsBotIDs: fetchAppsBotIDs,
```

Comment:
0/5 Maybe name the prop `fetchAppsBotIDs` so it's more clear what's happening within the component's code